src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -100,19 +100,19 @@ public String beforeOrAfter(LocalDate someDate) {
         LocalDate now = LocalDate.now();
         if (someDate.isAfter(now)) {
             return someDate.toString() + " is after " + LocalDate.now();
-        } else if (someDate.equals(now)) {
+        }
+        if (someDate.equals(now)) {
             return someDate.toString() + " is today";
```

Comment:
можна не використовувати toString() з датою)
```suggestion
            return someDate + " is today";
```